apps/met/themis-fe/themis-fe.yaml

```diff
@@ -12,7 +12,7 @@ spec:
       aadpodidbinding: met-schema-binding
     nodejs:
       ingressHost: 'gobgateway.test.platform.hmcts.net'
-      image: 'sdshmctspublic.azurecr.io/themis/themis_sb_azure:1.0.0'
+      image: 'sdshmctspublic.azurecr.io/themis/themis_sb_azure:1.0.1'
```

Comment:
Maybe worth us talking about automated image deployments rather than manual PR each time for image update

Reply:
Discussed with CGI.